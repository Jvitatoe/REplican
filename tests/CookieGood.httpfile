Content-type: text/html
Set-Cookie: Test=Expires; Expires=Sat, 10-May-2028 21:15:19 GMT

<html>
<head>
<title>Test Expires</title>
</head>
<body>
</body>
</html>
