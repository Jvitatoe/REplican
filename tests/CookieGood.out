# HTTP Cookie File
# This is a generated file!  Do not edit.
localhost	FALSE	Cookies/CookieGood.httpfile	FALSE	1841606119	Test	Expires
